packages/spacecat-shared-aws-api-client/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@adobe/spacecat-shared-aws-api-client",
```

Comment:
are you building a generic AWS api client or just a cost-explorer? depending on the answer the package name needs adjusting.

if you're building a generic AWS api client, what would its features be?